### ISSUE TITLE
@disable setting have not effect on project level

### ISSUE THREAD

**Issue by Bnaya** [SublimeLinter/SublimeLinter#378]
I have sublimelinter +sublimelinter-jshint installed

I want to disable sublimelinter-jshint for spesific project, i've tired to disable it via the project settings and .sublimelinterrc and its not working

my project file:

``` json
{
    "folders":
    [
        {
            "path": "."
        }
    ],
    "Sublimelinter":
    {
        "linters": {
            "jshint": {
                "@disable": true,
                "args": [],
                "excludes": []
            }
        }
    }
}
```

.sublimelinterrc:

``` json

{
    "linters": {
        "jshint": {
            "@disable": true,
            "args": [],
            "excludes": []
        }
    }
}

```

**Comment by ajmichels** [comment]
Might be helpful if you provided the contents of your SublimeLinter settings file

Sublime Text > Preferences > Package Settings > SublimeLinter > Settings - User

**Comment by FichteFoll** [comment]
I tried disabling the flake8 linter for a specific folder (not the entire project) using `.sublimelinterrc` but the settings seems not to be honored. Here are its contents:

```json
{
    "linters": {
        "flake8": {
            "@disable": true
        }
    }
}
```

I also tried inline settings a la `# [SublimeLinter flake8-@disable:true]` but that doesn't have any effect either.

Following that, I enabled debug settings but there are no mentions about a `.sublimelinterrc` file or inline settings found. I don't know if there is debugging output for those at all, so I can't tell if they're not getting found or if something's up internally.
